crypto/vrf/vrf_r2ishiguro_test.go

```diff
@@ -10,6 +10,31 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestVrfEd25519r2ishiguro_ProofToHash(t *testing.T) {
```

Comment:
Like the next function, could you name this `TestVrfEd25519[R]2ishiguro_ProofToHash()` since `r2ishiguro` is one word?

Reply:
I'll fix this. Thanks for checking!